extensions/amp-story-audio-sticker/0.1/amp-story-audio-sticker.js

```diff
@@ -157,14 +159,69 @@ export class AmpStoryAudioSticker extends AMP.BaseElement {
       : FALLBACK_DEFAULT_STICKER;
   }
 
+  /**
+   * @private
+   */
+  maybeInitializeStickerStyle_() {
+    this.toggleStickerState_(
+      'outline',
+      this.element.getAttribute('sticker-style') === 'outline'
+    );
+    this.toggleStickerState_(
```

Comment:
I wonder if you even need this sticker state. You can get the value in CSS with `[sticker-style="dropshadow"] {}` instead of adding a new class.

Reply:
True. Done.